Woof/Woof/ViewModel/OwnerMainTabViewModel.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+/// The view model for a owner main tab view, responsible for preparing and processing data for the view.
+final class OwnerMainTabViewModel: ObservableObject {
+    /// Resets the current owner role in the app by default value `none`.
```

Comment:
Does it sound more clear?
```suggestion
    /// Resets the current user role in the app to the default value.
```

Reply:
Yes, thanks. Fixed it.